packages/fxa-auth-server/lib/senders/emails/layouts/fxa/index.txt

```diff
@@ -0,0 +1,9 @@
+<%- body %>
+
+Mozilla. 2 Harrison St, #175, San Francisco, CA 94105
+
+fxa-privacy-url = "Mozilla Privacy Policy"
+<%- privacyUrl %>
+
+fxa-support-url = "Firefox Cloud Terms of Service"
+<%- supportUrl %>
```

Comment:
I have added these new text files to generate plaintext version with the new templates. A pattern is set: id = "value" in these text files for fluent to get the localized string. The idea is that if the `id` doesn't exit in the fluent bundle than it will fall back to using the default value present in these text files i.e `val`. Let me know if this approach needs to be further worked upon

Reply:
This seems clever to me! Maybe we can tweak it later if we run into any edge cases.